WORKSPACE.bazel

```diff
@@ -15,11 +15,11 @@ http_archive(
 
 # File
 
-FILE_VERSION = "fd7ad6a7e9f3438fddb41683f33b6b4d07d067fe"
+FILE_VERSION = "7e07245edcc54d419a52d141b3c8e86e3dc0d5c5"
 
 http_archive(
     name = "rules_file",
-    sha256 = "9108bf11fe1063bb6268f15506ea35c1a29e03cae651032d1da3d0f33561c5fd",
+    sha256 = "ee31f2ebcd09e289090420012d19ffaecb71c005e74f511301f1fd47cbd5f971",
     strip_prefix = "rules_file-%s" % FILE_VERSION,
     url = "https://github.com/hello-seer/rules_file/archive/%s.zip" % FILE_VERSION,
```

Comment:
We have rules_javascript's rules_file pulling from a fork of rivet's rules_file.  Feels kind of messy, what's going on?

Reply:
Ah, didn't realize, we just need to put those changes back into Rivet's.

I'll look at that.

Reply:
The changes were already in rivethealth/rules_file. I updated the URL to use that.